src/main/resources/application.properties

```diff
@@ -9,8 +9,8 @@ spring.jpa.hibernate.ddl-auto: update
 spring.jpa.show-sql:true
 spring.jpa.properties.hibernate.format_sql: false
 #SENDGRID
-app.sendgrid.key:
-app.sendgrid.from-address:
+app.sendgrid.key: ${SENDGRID}
```

Comment:
Please add those variables to the ReadMe.

Reply:
done